solutions/cpython-tests/test_config_v3.9.7/patch

```diff
@@ -166,6 +166,14 @@ index f4edb8b..924582c 100644
      def test_sched_rr_get_interval(self):
          try:
              interval = posix.sched_rr_get_interval(0)
+@@ -1469,6 +1480,7 @@ class PosixTester(unittest.TestCase):
+         open(fn, 'wb').close()
+         self.assertRaises(ValueError, os.stat, fn_with_NUL)
+ 
++    @unittest.skip("Temporarily disabled in Mystikos")
```

Comment:
Add reason - SYS_pidfd_open not supported in Mystikos.
Also do we need to skip for cpython 3.8?

Reply:
It fails only in the 3.9 case